tvb_contrib/tvb/contrib/tests/cosimulation/parallel/modify_wongwang_precision_test.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+#
+#
+#  TheVirtualBrain-Contributors Package. This package holds simulator extensions.
+#  See also http://www.thevirtualbrain.org
+#
+# (c) 2012-2020, Baycrest Centre for Geriatric Care ("Baycrest") and others
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software Foundation,
+# either version 3 of the License, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License along with this
+# program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+#   CITATION:
+# When using The Virtual Brain for scientific publications, please cite it as follows:
+#
+#   Paula Sanz Leon, Stuart A. Knock, M. Marmaduke Woodman, Lia Domide,
+#   Jochen Mersmann, Anthony R. McIntosh, Viktor Jirsa (2013)
+#       The Virtual Brain: a simulator of primate brain network dynamics.
+#   Frontiers in Neuroinformatics (7:10. doi: 10.3389/fninf.2013.00010)
+
+"""
+.. moduleauthor:: Lionel Kusch <lkusch@thevirtualbrain.org>
+.. moduleauthor:: Dionysios Perdikis <dionperd@gmail.com>
+"""
+
+import numpy as np
+
+import tvb.simulator.lab as lab
+from tvb.tests.library.base_testcase import BaseTestCase
+from tvb.contrib.tests.cosimulation.parallel.ReducedWongWang import ReducedWongWangProxy
+
+from tvb.contrib.cosimulation.cosim_monitors import RawCosim
+from tvb.contrib.cosimulation.cosimulator import CoSimulator
+
+
+SIMULATION_LENGTH = 3.0
```

Comment:
why is this a global? why not a method arg below?

Reply:
I add it in the attribute of the test class.
It's not an argument of _reference_simulation because it's need to in the test for some of assertions.